pkg/k8s/serviceAccountManager_test.go

```diff
@@ -90,3 +90,20 @@ func Test_CreateRoleOtherNamespace_works(t *testing.T) {
 	_, err := acc.AddRoleBinding(roleName, ns1)
 	assert.NilError(t, err)
 }
+
+func Test_SetDoAutomountServiceAccountToke_works(t *testing.T) {
+	//SETUP
+	fakeServiceAccount := fakeServiceAccount()
+	setupAccountManager(fakeServiceAccount)
+	acc, err := accountManager.GetServiceAccount(accountName)
+	assert.NilError(t, err)
```

Comment:
Also check if `*acc.cache.AutomountServiceAccountToken` is NOT already false by default. Otherwise you would not know if the Set worked.